src/Analysis/GWASV2/Shared/AttritionTableWrapper/AttritionTableWrapper.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AttritionTable from './AttritionTable/AttritionTable';
+
+const AttritionTableWrapper = ({
+  newCovariateSubset,
+  selectedCohort,
+  otherSelectedCohort,
+  outcome,
+  sourceId,
+}) => (
+  <React.Fragment>
+    <AttritionTable
+      sourceId={sourceId}
+      newCovariateSubset={newCovariateSubset}
+      selectedCohort={selectedCohort}
+      otherSelectedCohort={otherSelectedCohort}
+      outcome={outcome}
+      // selectedCovariates={selectedCovariates}
+      // selectedDichotomousCovariates={selectedDichotomousCovariates}
+      tableHeader={'Case Cohort Attrition Table'}
```

Comment:
we should start with just "Attrition Table". 
In step 2, once the outcome is known, and _only if_ it is `custom_dichotomous`, then we do the following:
- set tableHeader of first table to `{'Case Cohort Attrition Table'}`
- set tableHeader of second table to `{'Control Cohort Attrition Table'}`

Reply:
Interesting!

Updated to 

```
const useSecondTable = outcome.variable_type === 'custom_dichotomous';

```

And then for the first table

```
       tableHeader={
          useSecondTable ? 'Case Cohort Attrition Table' : 'Attrition Table'
        }
```